zerver/tests/test_events.py

```diff
@@ -1307,16 +1305,14 @@ def test_change_desktop_icon_count_display(self) -> None:
             lambda: do_change_notification_settings(
                 self.user_profile,
                 notification_setting,
-                2,
-                log=False))
+                2))
```

Comment:
We want as a policy to always pass `acting_user`, so I changed that in these test call points.